src/audits/manifest/short-name.js

```diff
@@ -49,8 +50,11 @@ class ManifestShortName extends Audit {
     let hasShortName = false;
     const manifest = artifacts.manifest.value;
 
-    if (manifest && manifest.short_name) {
-      hasShortName = (!!manifest.short_name.value);
+    // putting a comment to test travis again
+    if (manifest) {
+      const manifestShortName = _get(manifest, 'short_name.value');
+      const manifestName = _get(manifest, 'name.value');
+      hasShortName = !!(manifestShortName || manifestName);
```

Comment:
yep once _get is dropped this will be what we want.. right now it is working due to _get